rust/candid/src/bindings/mod.rs

```diff
@@ -2,19 +2,3 @@
 // This module assumes the input are type checked, it is safe to use unwrap.
 
 pub mod candid;
```

Comment:
Can we try to remove the `bindings` modules, and make `candid` module as something like `pretty_printer`?